Finding Eigen3 breaks on a standard installation

I get:
```
CMake Error at CMakeLists.txt:39 (FIND_PACKAGE):
  Could not find a package configuration file provided by "Eigen3" with any
  of the following names:

    Eigen3Config.cmake
    eigen3-config.cmake

  Add the installation prefix of "Eigen3" to CMAKE_PREFIX_PATH or set
  "Eigen3_DIR" to a directory containing one of the above files.  If "Eigen3"
  provides a separate development package or SDK, be sure it has been
  installed.
```
However, I have:
```
/usr/share
/usr/share/cmake
/usr/share/cmake/Modules
/usr/share/cmake/Modules/FindEigen3.cmake
/usr/share/pkgconfig
/usr/share/pkgconfig/eigen3.pc
```
installed just fine. 
Modifying `CMakeLists.txt` from:
```
FIND_PACKAGE(Eigen3 REQUIRED NO_MODULE)
```
to
```
FIND_PACKAGE(Eigen3 REQUIRED)
```
fixes that:
```
-- Found Eigen3: /usr/include/eigen3 (Required is at least version "2.91.0") 
```
I wonder why `NO_MODULE` is specified explicitly here, since it breaks any standard installation. 
Also, in case there is no module, a fallback to "config mode" will be done in any case. 

Is there a reason for it (then all distros out there need to patch GenFit), or can it be dropped for good?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,7 +36,7 @@ IF(${ROOT_found_version} LESS 60000)
 	MESSAGE(FATAL_ERROR "GenFit requires a ROOT version > 6.00.00")
 ENDIF()
 
-FIND_PACKAGE(Eigen3 REQUIRED NO_MODULE)
+FIND_PACKAGE(Eigen3 REQUIRED)
 
 # either, the environment variable RAVEPATH has to be specified, and RAVE's pkgConfig will be used to determine everything we need
 # or, RAVE_LDFLAGS, RAVE_INCLUDE_DIRS and RAVE_CFLAGS have to be set via the command-line

